src/pages/product-list-page/product-list-item/product-list-item.js

```diff
@@ -44,9 +44,13 @@ const ProductListItem = ({ product }) => {
     );
     return availableSizes && availableSizes[0] ? (
       <>
```

Comment:
Why do you need this Fragment here?

Reply:
if you mean point 46, I just forgot to delete it, I'll do it now